orchestra/static/orchestra/todos/todo-qa.directive.es6.js

```diff
@@ -14,8 +14,38 @@ export default function qa () {
     controller: function (todoApi, todoQaApi, $scope) {
       var todoQa = this
       todoQa.todos = []
+      todoQa.project = null
       todoQa.ready = false
 
+      todoQa.copyToClipboard = str => {
+        const el = document.createElement('textarea')
+        el.value = str
+        document.body.appendChild(el)
+        el.select()
+        document.execCommand('copy')
+        document.body.removeChild(el)
+      }
+
+      todoQa.commentSummary = (todos, summary) => {
+        summary = summary || ''
+        todos.forEach((todo) => {
+          summary = todoQa.commentSummary(todo.items, summary)
+          if (todo.qa && !todo.qa.approved && todo.qa.comment) {
```

Comment:
what if you leave a comment of praise on an approved todo? presumably you don't want to filter on `!todo.qa.approved`?

Reply:
I thought about it. I used only disapproved todos to keep it consistent with the logic for showing feedback on their todos.